source-code/ide-extension/package.json

```diff
@@ -28,9 +32,23 @@
 			}
 		]
 	},
+	"typesVersions": {
+		"*": {
+			"plugin": [
+				"./dist/plugin/index.d.ts"
+			]
+		}
+	},
```

Comment:
@samuelstroschein How do we publish the ide-extension plugin?

Reply:
Just as we publish other plugins, directly via github. 

But, other plugins should build on top of the ide extension plugin. Thus, publishing to npm might be a good idea after all. We can't use `@inlang/ide-extension` as a namespace though because vscode doesn't support `@organization` syntax. 

Might be easier to have two separate package.json files/projects for the ide extension and the ide extension plugin? Should also ease bundling and typing things as the plugin-template can be used. 

```
source-code/ide-extension-plugin [@inlang/plugin-ide-extension] -> this would need to be imported by developers
source-code/ide-extension [vs-code-extension]
```

Reply:
@samuelstroschein or even it's own repository like the other plugins / standard lint rules etc? 

then it's really clear that this is some standalone plugin code. 

published under the inlang organization (not my account).

Reply:
@samuelstroschein off topic here but regarding the ide extension marketing in the marketplace I would suggest something more common like translation intellisense or something with -"lens", 
-> inlang - i18n translation intellisense, not only "inlang"

we can A/B test this once it's out.

Reply:
@samuelstroschein I have moved it into it's own folder for now. can be further extracted.